dev/bots/test.dart

```diff
@@ -153,8 +153,12 @@ bool _shouldRunWindows() {
 /// Verify the Flutter Engine is the revision in
 /// bin/cache/internal/engine.version.
 Future<void> _validateEngineHash() async {
+  final String overrideFlag = Platform.environment['FLUTTER_SKIP_ENGINE_CHECK']
+      ?? 'false';
+  // TODO(karlklose): remove the bot name check once the 3H builder recipe uses
+  //                  FLUTTER_SKIP_ENGINE_CHECK.
```

Comment:
We could also just make that change already, adding the extra variable shouldn't have any side effects until this lands. Then this PR wouldn't need the TODO and could remove the hardcoded bot prefix `luci-dart-` already now.

Reply:
Done.